frontend/components/Home.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Home = () => {
+
+    return (
+        <>
+            <h2>Welcome to Text-Based Sentiment Analysis!</h2>
```

Comment:
Unless there's another `<h1>` being generated somewhere, we should rewrite this as an `<h1>`. Per the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements), HTML headers should follow the hierarchy indicated by their number (i.e., we shouldn't skip an `<h1>` to go straight to an `<h2>`).

Reply:
Ah yes! I put an `<h1>` tag in `Nav.js` surrounding the `Sonification` link. I wasn't sure if that made sense!